index.bs

```diff
@@ -1321,6 +1590,57 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
 
 	</div><!-- algorithm -->
 
+	<div class="algorithm" data-algorithm="write-blobs-to-clipboard">
+	<h3 id="to-write-blobs-to-clipboard"><dfn>write blobs and option to the clipboard</dfn></h3>
+
+		: Input
+		:: |items|, a [=sequence=]&lt;{{Blob}}&gt; list of {{Blob}}s to write
+		:: |option|, a [=ClipboardItem/clipboard item=]'s [=presentation style=]
+
+		: Output
+		:: None
+
+		1. If the |items| list is not empty, then
+
+			1. For each |item| in |items|:
+
+				1. If data type is <i>text/plain</i>, then
+
+					1. Ensure encoding is correct per OS and locale conventions
+
+					1. Normalize line endings according to platform conventions
+
+					1. Place text on clipboard with the appropriate OS clipboard
```

Comment:
This seems bad; it should be done in terms of [=clipboard items=] and representations and such. writeText() has good algorithms for this sort of thing!

Reply:
This is the part that interacts with the system clipboard so not sure if I can use [=clipboard items=] to write contents to the OS clipboard service. I'll add some more details about OS specific formats and write methods.